### PR TITLE
First matching improvement

### PR DESCRIPTION
**Pull request by michaloo**

### CODE REVIEW

File: server/lib/custom-audiences.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import crypto from "crypto";
+
+/**
+ * @see https://developers.facebook.com/docs/marketing-api/custom-audience-api/v2.8
+ */
+export default class CustomAudiences {
```
Comment (sbellity) [inline server/lib/custom-audiences.js]: this does not really need to be a class, can we set it up as a simple module instead ?

Comment (michaloo) [inline server/lib/custom-audiences.js]: Sure. That was just a quick try. I will simplify thay.

Comment (michaloo) [inline server/lib/custom-audiences.js]: I've added here second public method. Do you agree it will stay as a class?

**Comment by michaloo** [comment]
@sbellity I've done most of the work, upgraded the hull-node lib and did some code cleanup (moved in two `util` modules). Wanna have a look?

**Pull request merged**